docusaurus/docs/what-is-included/custom-error-page.md

```diff
@@ -0,0 +1,22 @@
+---
+id: custom-error-page
+title: Custom Error Page
+sidebar_label: Custom Error Page
+---
+
+The boilerplate supports a minimalistic error page that is able to distinguish between `Page Not Found` and `Internal Server Error`. It contains appropriate copy for each case, and a link to `/`.
+
+Adapting the ErrorPage component
+---
+
+You should update this page to fit in with your project's image. To do so, you will need to update the following files:
+
+```
+www
+├── pages
+│   ├── error
+│   │   ├── ErrorPage.js
```

Comment:
Nifty pick: to keep consistency, shouldn’t this be named just `Error`?

Reply:
I've gone back and forth on this one. I wanted to avoid any ambiguity with javascript's `Error`. Named the files `ErrorPage.*` to match the name of the actual component

Reply:
I see. Let's go with ErrorPage then.